modules/core/src/v2/coins/trx.ts

```diff
@@ -0,0 +1,49 @@
+const tronweb = require('tronweb');
```

Comment:
Is there a types package for this lib?

Reply:
there is not, unfortunately

Reply:
i lied. there's a d.ts file out there. I don't think it's official but @dpc found something

Reply:
after talking about it, the typings file is not complete, we'll probably only push typings for the things we use